core/akka-runtime/src/main/scala/com/ing/baker/runtime/akka/actor/process_instance/ProcessInstanceEventSourcing.scala

```diff
@@ -52,6 +52,21 @@ object ProcessInstanceEventSourcing extends LazyLogging {
                                    failureReason: String,
                                    exceptionStrategy: ExceptionStrategy) extends TransitionEvent
 
+  /**
+    * An event describing the fact that a transition has been postponed to be fired.
+    */
+  case class TransitionDelayed(override val jobId: Long,
```

Comment:
`TransitionDelayed ` and `DelayedTransitionFired` have identical Javadoc. I don't think that's correct?